extension/src/popup/views/Account/styles.scss

```diff
@@ -1,9 +1,9 @@
 :root {
-  --account-view-padding-left: 1.5rem;
+  --account-view-padding-side: 1.5rem;
```

Comment:
took me a solid min to figure out why we were using only `left` for the side padding 😆  so I changed to `side`